pyproject.toml

```diff
@@ -1,8 +1,8 @@
 [build-system]
 build-backend = "setuptools.build_meta"
 requires = [
-  "setuptools>=65",
-  "setuptools-scm[toml]>=6.2",
+  "setuptools>=64",
```

Comment:
Why lowering the version here?

Reply:
Sync with cta pyproject and ctapipe. I don't remember why we had 65 (and I don't know why 64). I trust the guys responsible for those package know more than me....

Reply:
Ok good